control-plane/pkg/reconciler/base/receiver_condition_set.go

```diff
@@ -51,6 +52,7 @@ var IngressConditionSet = apis.NewLivingConditionSet(
 	ConditionConfigMapUpdated,
 	ConditionConfigParsed,
 	ConditionProbeSucceeded,
+	ConditionEventPoliciesReady,
```

Comment:
I'm pretty sure that since this condition set is used by all components that receive events, we will need to set this condition to some form of "ready" value in the kafka sink and kafka channel reconciler code so that the tests can deploy properly

Reply:
Good point. I updated it for KafkaSink. KafkaChannel seems to set its own ConditionSet (which we then need to update, when we add the support in the KafkaChannel): https://github.com/knative-extensions/eventing-kafka-broker/blob/073bcd241e318b3c8e7a46e875e688025a32decd/control-plane/pkg/reconciler/channel/controller.go#L67